### PR TITLE
DockerHub Image Documentation

### PR DESCRIPTION
**Pull request by dmccoystephenson**
## Changes
Documentation for using the image has been added to `docs/dockerhub.md`.

### CODE REVIEW

**Review by drewjj — APPROVED** [review]
This looks good to me.

File: docs/dockerhub.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
+# jpo-s3-deposit
+
+## GitHub Repository Link
+https://github.com/usdot-jpo-ode/jpo-s3-deposit
+
+## Purpose
+The purpose of the jpo-s3-deposit program is to deposit messages to an Amazon S3 bucket or some other destination.
```
Comment (payneBrandon) [inline docs/dockerhub.md:7]: Maybe tweak a bit to something like:
`
The purpose of the jpo-s3-deposit program is to deposit messages to permanent storage, such as an Amazon S3 bucket, mongodb instance, or other similar system.
`

**Review by payneBrandon — CHANGES_REQUESTED** [review]
Just the one comment

**Review by payneBrandon — APPROVED** [review]
lgtm!

**Pull request merged**